andela/wger-prometheus
# #150473789 Fix failing tests and PEP8 styling

### What does this PR do?
Fix failing tests and fix PEP8 styling issue with almost all files.

### How Do I manually test this?
After installing the project, run the command
`./manage.py test`
1093 tests should run without error

### Relevant Pivotal Tracker Story
[150473789](https://www.pivotaltracker.com/story/show/150473789)

Screen Shot if appropriate
<img width="393" alt="screen shot 2017-08-25 at 14 46 31" src="https://user-images.githubusercontent.com/6042152/29712795-53c55890-89a4-11e7-8ec1-a4bf94a7fa2b.png">
